builtin/common/misc_helpers.lua

```diff
@@ -685,6 +685,12 @@ end
 -- Returns the exact coordinate of a pointed surface
 --------------------------------------------------------------------------------
 function core.pointed_thing_to_face_pos(placer, pointed_thing)
+	-- Avoid crash when head is inside a non-walkable and buildable_to node,
```

Comment:
Not necessary a non-walkable and buildable_to node. It can be any node if the player placed into themselves (minetest.conf setting).

Reply:
Ah ok, will update.

Reply:
In testing, it doesn't crash when inside a normal cube node, using 'build where stand'. However will make comment less specific.

Reply:
Updated.